trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/get-back-link-href.ts

```diff
@@ -0,0 +1,31 @@
+// "Recommended Security and Performance Max: 2048 CHARACTERS" - https://stackoverflow.com/a/48230425
+const MAX_URL_CHARACTERS = 2048;
+const BASE_URL_CHARACTER_ALLOWANCE = 150;
+
+export const getAddToExistingPaymentBackLinkToUtilisationReportReconciliationPage = (reportId: string, feeRecordIds: number[]): string => {
+  const urlWithoutParams = `/utilisation-reports/${reportId}`;
+
+  if (feeRecordIds.length === 0) {
```

Comment:
Seems we should move the `urlWithoutParams` const into this condition

Reply:
We have further references to it further down the function (eg to return when we exceed the URL character limit) so keeping as is

Reply:
Sorry didn't spot this